pkg/data/management/kontainerdriver_data.go

```diff
@@ -92,7 +92,7 @@ func addKontainerDrivers(management *config.ManagementContext) error {
 	}
 	if err := creator.addCustomDriver(
 		"oraclecontainerengine",
-		"https://github.com/rancher-plugins/kontainer-engine-driver-oke/releases/download/v1.6.0/kontainer-engine-driver-oke-linux",
+		"https://github.com/rancher-plugins/kontainer-engine-driver-oke/releases/download/v1.7.0/kontainer-engine-driver-oke-linux",
```

Comment:
If the version is getting updated the hash needs to be updated as well, for an example: https://github.com/rancher/rancher/commit/5fd46053130e52a60576ba0b00f2bb070f38b600